### DISCUSSION TITLE
How to test only specific test cases(exist in different paths) which are more than 100 using CLI

### DISCUSSION THREAD

**Discussion by SwardCoder** [cypress-io/cypress#24067] [Questions and Help]
Hi Team 
is there any way I want to pass the paths of the test cases which i wish to run in a file or dictionary 
and run them via cypress CLI

Example 
File1.txt
-- abc/cd/Doc1.spec.ts
-- xyz/sa/Test.spec.ts
-- cvd/dc/Testing.spec.ts

and i want to run all the specs which are in the text file  via CLI 

i dont want to make CLI manually like 
npx cypress run --spec="abc/cd/Doc1.spec.ts, xyz/sa/Test.spec.ts ..........."
because if there are 1000 test cases failing then its a problem for me with specific string if i wish to use above approach 

so please help me with your valuable ideas 
thank you 🙂 

the intension behind this is if i get some 3 test cases failing i will maintain them in a file 
then i will ask developers to fix the appropriate things then once they fix it 
i would like to run only this first 
Note: i would consider failing of this test cases even after using retries option for flaky test cases

**Comment by mkonev-casafari** [comment]
Hello, 

pls take a look on following plugins

https://github.com/bahmutov/cypress-select-tests
https://github.com/cypress-io/cypress-grep
